_posts/2017-05-10-april.md

```diff
@@ -0,0 +1,86 @@
+
+I'd like to say thinks to everyone who gave their input regarding the syntax. When we released it with v0.2, we didn't know how it would be received, so we were happy to see that people liked it enough to request it return with v0.3. As always, we're developing Eve in the open and with our community, so your feedback has a big impact on how we prioritize work.
+
+### Web Presence
+
+Before we can officially release v0.3, a lot of work needs to be done on the usability and documentation side of things. To this end, we revamped our website to make it more accessible to people unfamiliar but curious about Eve. A big part of this effort was actually coming up with a one-sentence pitch, describing Eve in a nutshell.
+
+#### A new pitch for Eve
+
+Here's our new one sentence pitch for Eve:
+
+> Eve is a modern relational language for writing data-driven programs without the boilerplate.
+
+The intended audience for this pitch is generally the "HackerNews" crowd -- developers who are curious about languages and likely to something new to the scene. Eve has many aspirations (being a language for non-programmers), but this pitch is directed more toward explaining Eve in its current form to programmers. We wanted to pack as much information into this one sentence as possible, so you can get a general idea of the concepts important to Eve after reading it. This is actually easier said than done, so I'll break down how we went about this.
+
+The first thing we did was look at how other languages described themselves. We picked about a half dozen languages and chose our favorite pitches between them. Some, like Python, had very nebulous pitches: "Python is a programming language that lets you work quickly and integrate systems more effectively." I think for anyone whose used Python, this is an accurate description of the language, but it doesn't really say how Python lets you work quickly. Others, like Haskell, were very specific: "An advanced, purely functional programming language". That's definitely Haskell in a nutshell.
+
+We wanted our pitch to tell the reader what Eve is, what makes it unique, and why one would use it over other languages. From the pitch, you can see we arrived at these traits
+
+**Modern** - It was important for us to note that Eve is a new take on programming, specifically on "relational languages". On its own, relational may connote SQL, which is not what we want to do. By claiming that we are "modern" we hope to tell the reader that Eve is an updated approach to things they may have seen in the past.
+
+**Relational language** - there were several candidates to describe what *kind* of language Eve is. Pattern matching, logic, declarative. We decided on "relational" because relations are a concept central to programming in Eve, and we wanted to highlight this fact up-front.
```

Comment:
up-front should not be hyphenated because it's being used as preposition not an adjective